Servers/Zeus/ARC/.arc/submit.py

```diff
@@ -34,7 +34,11 @@
 
 cp "$PBS_O_WORKDIR/input.gjf" $GAUSS_SCRDIR
 
-cp "$PBS_O_WORKDIR/check.chk" $GAUSS_SCRDIR
+if [ ! -f check.* ]; then
+	:
```

Comment:
Shouldn't it be the other way around - if the check file doesn't exist in the scratch directory, then copy it there?